src/common/boards/nrf_PCA10175_1.0.0_54H20.json

```diff
@@ -0,0 +1,111 @@
+{
+    "board": {
+        "boardVersion": "PCA10175",
+        "boardRevision": "1.0.0",
+        "boardName": "nRF54H20 DK v1.0.0"
+    },
+    "pins": [
+        {
+            "type": "vcom",
+            "id": "vcom0",
+            "name": "VCOM0",
+            "enable": {
+                "pin": 42
+            },
+            "hwfc": {
+                "pin": 20
+            }
+        },
+        {
+            "type": "vcom",
+            "id": "vcom1",
+            "name": "VCOM1",
+            "enable": {
+                "pin": 22
+            },
+            "hwfc": {
+                "pin": 23
+            }
+        },
+        {
+            "type": "switch",
+            "id": "swd3-control",
+            "title": "Software Debugger (SWD)",
+            "label": "Disable to use an external debugger on the target chip.",
+            "tooltip": "When enabled, the SWD is connected and the IMCU is acting as the debugger. When disabled, the SWD is disconnected and you can use an external debugger connected to the Debug In header.",
+            "enable": {
+                "pin": 6
+            }
+        },
+        {
+            "type": "switch",
+            "id": "led-pwr-ctrl",
+            "title": "LED Power",
+            "label": "Disable LEDs to use the GPIOs for other purposes.",
+            "tooltip": "The LEDs are enabled by providing 5 V to LED0-LED3.",
+            "enable": {
+                "pin": 45
+            }
+        },
+        {
+            "type": "switch",
+            "id": "ldos-flash-dis",
+            "title": "External Memory",
+            "label": "Enable power to the external flash memory.",
+            "tooltip": "Enable external flash by powering the rails VCCQ_1V8 and VCC_1V8.",
+            "enable": {
+                "pin": 47,
+                "invert": true
+            }
+        }
+    ],
+    "pmicPorts": [
+        {
+            "type": "voltage",
+            "port": 1,
```

Comment:
was port: 2, but I assume this is just an index property?

Reply:
Port number refers to actual port in the PMIC, and is not arbitrary. Changing port here will alter another output on the PMIC.

Reply:
It says to remove controls for ports 1,2 and 7. 
As for ports numbers we have only 1-4 range, I've removed controls based on their descriptions as that might be what user see. Is that correct?

For the port property will return 2 number

Reply:
The GPIO ports are "groups" of electronic pins on the target microcontroller (54H20) itself. The port number here is the output on the power management chip (supplying different voltages to the 54H20 which is, in turn, used to power the GPIO ports).

To find out which PMIC port we need to hide, we need to look into the HW design of the DK:
https://nordicsemi.atlassian.net/wiki/spaces/APPS/pages/67295432/nRF5340+Board+Controller+-+1.6+nRF5340+Pinout#nRF54H20-DK

![image](https://github.com/user-attachments/assets/5468244e-8735-433e-831b-e41aa71ad6e5)

From the table, it looks like only port 2 (GPIO port 9) should be configurable, leaving the three other IO ports out.

This means we should have only port 2 in the definition file.